src/CommunityToolkit.Maui.UnitTests/Views/UniformGrid_Tests.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CommunityToolkit.Maui.UnitTests.Mocks;
+using CommunityToolkit.Maui.Views;
+using Microsoft.Maui;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Graphics;
+using NSubstitute;
+using Xunit;
+
+namespace CommunityToolkit.Maui.UnitTests.Views;
+
+public class UniformGridTests : BaseTest
+{
+	const double _childCount = 3;
+	const double _childWidth = 20;
+	const double _childHeight = 10;
+
+	readonly UniformGrid _uniformGrid;
+	readonly IView _uniformChild;
+
+	public UniformGridTests()
+	{
+		_uniformChild = Substitute.For<IView>();
```

Comment:
Can we accomplish this Unit Test without NSubstitute? 

I'd prefer that we test against the actual public APIs instead of creating substitutes.